nyxx/lib/src/utils/CacheUtility.dart

```diff
@@ -0,0 +1,113 @@
+part of nyxx;
+
+/// Cache Utility is a Utility that lets you retrieve Entities (from the cache) from outside of the nyxx project. Typically used in nyxx.* projects.
+/// An example getting a user is below:
+/// ```dart
+/// void main() {
+///     var bot = Nyxx("TOKEN");
+///     Cacheable<Snowflake, User> cachedUser = CacheUtility.users(bot, Snowflake(''));
+/// }
+/// ```
+class CacheUtility {
+  /// Retrieves a cached User.
+  /// ```dart
+  /// void main() {
+  ///     var bot = Nyxx("TOKEN");
+  ///     Cacheable<Snowflake, User> cachedUser = CacheUtility.users(bot, Snowflake(''));
+  /// }
+  /// ```
+  Cacheable<Snowflake, User> users(
```

Comment:
better name would be something like `createCacheable` because now name doesnt tell much

Reply:
Fixed in next push 👍🏼